### DISCUSSION TITLE
Maximum residual point and total time missing in v7?

### DISCUSSION THREAD

**Discussion by timjim333** [su2code/SU2#1136] [General]
Dear Team,

Firstly, thanks for the latest SU2 update! I noticed on migrating from v6 to v7 that the maximum residual location is no longer displayed in the screen output. Is there any way to get this information back or exported to the history file? I tied looking through the `SU2_CFD -d` options but it didn't look like there was an output for this. I found it to be quite useful for debugging mesh issues.

Secondly, is there an equivalent to the v6 wall clock Time(min) output for the history file? I tried `WALL_TIME` but this gave the individual iteration times instead. I assume that cumulatively adding these output might give an approximation of the runtime but may be less than the actual time due to I/O and overhead?

Thanks and kind regards,
Tim

**Comment by pcarruscag** [comment]
Indeed that output fell through the cracks, the info is still computed though... Maybe we can make it one of the history outputs.

WALL_TIME is the average time per iteration, so multiply the final value by the number of iterations +1 to get the total.
You can also use the WRT_PERFORMANCE option (=YES) which will print more info.

**Comment by timjim333** [comment]
I think it would be great to output it to the history. I guess an alternative I can see at the moment is to add the `RESIDUAL` group to the `VOLUME_OUTPUT` flag and then post-process, but for a quick glance at problematic meshes, a history listing would be easier!

Regarding WALL_TIME, this changes every iteration. Would summing it be sufficient? Or would this be less than the actual run time? Thanks and regards,
Tim

**Reply by koodlyakshay** [reply]
The WRT_PERFORMANCE option should give you the summary of average time per iteration, total time and so on. The summary gets written at the end of your run.

**Reply by timjim333** [reply]
Thanks for the hint 😊

**Reply by timjim333** [reply]
@pcarruscag Regarding the total wall time, the problem with using the WRT_PERFORMANCE over having it in the history file as in v6 is that it makes it harder to track down total times after restarts, as you need to scan back through previous screen outputs. As I'm interested in optimising my meshes for multi-case studies, having it in history was really helpful! Anyhow, if it's easy to put back in, that it. Thanks for all the dev work into SU2!
